components/widget/standings/widget_standings_swiss.lua

```diff
@@ -0,0 +1,140 @@
+function StandingsSwissWidget:render()
+	if not self.props.standings then
+		return
+	end
+
+	---@type StandingsModel
+	local standings = self.props.standings
+	local activeRounds = (Array.maxBy(
+		Array.filter(standings.rounds, function(round) return round.started end),
+		function (round) return round.round end
+	) or {round = 0}).round
+	local hasFutureRounds = not standings.rounds[#standings.rounds].started
+
+	return DataTable{
+		wrapperClasses = {'standings-ffa', 'toggle-area', 'toggle-area-' .. activeRounds},
+		classes = {'wikitable-bordered', 'wikitable-striped'},
+		attributes = {['data-toggle-area'] = activeRounds},
+		children = WidgetUtil.collect(
+			-- Outer header
+			HtmlWidgets.Tr{children = HtmlWidgets.Th{
+				attributes = {
+					colspan = 100,
+				},
+				children = {
+					HtmlWidgets.Div{
+						css = {['position'] = 'relative'},
+						children = {
+							HtmlWidgets.Span{
+								children = standings.title
+							},
+						},
+					},
+				},
+			}},
+			-- Column Header
+			HtmlWidgets.Tr{children = WidgetUtil.collect(
+				HtmlWidgets.Th{children = '#'},
+				HtmlWidgets.Th{children = 'Participant'},
+				HtmlWidgets.Th{children = 'Matches'},
+				Array.map(standings.rounds, function(round)
+					return HtmlWidgets.Th{children = round.title}
+				end)
+			)},
+			-- Rows
+			Array.flatMap(standings.rounds, function(round)
+				return Array.map(round.opponents, function(slot)
+					local positionBackground = slot.positionStatus and ('bg-' .. slot.positionStatus) or nil
+					local teamBackground
+					if not hasFutureRounds and slot.definitiveStatus then
+						teamBackground = 'bg-' .. slot.definitiveStatus
+					end
+					return HtmlWidgets.Tr{
+						children = WidgetUtil.collect(
+							HtmlWidgets.Td{
+								children = {slot.placement, '.'},
+								css = {['font-weight'] = 'bold'},
+								classes = {positionBackground},
+							},
+							HtmlWidgets.Td{
+								classes = {teamBackground},
+								children = OpponentDisplay.BlockOpponent{
+									opponent = slot.opponent,
+									showLink = true,
+									overflow = 'ellipsis',
+									teamStyle = 'hybrid',
+									showPlayerTeam = true,
+								}
+							},
+							HtmlWidgets.Td{
+								classes = {teamBackground},
+								children = table.concat({slot.matchWins, slot.matchLosses}, '-'),
+								css = {['font-weight'] = 'bold', ['text-align'] = 'center'}
+							},
+							Array.map(standings.rounds, function(columnRound)
+								local entry = Array.find(columnRound.opponents, function(columnSlot)
+									return Table.deepEquals(columnSlot.opponent, slot.opponent)
+								end)
+								if not entry then
+									return HtmlWidgets.Td{}
+								end
+
+								local match = entry.match
+								if not match then
+									return HtmlWidgets.Td{}
+								end
+
+								local opposingOpponentIndex = Array.indexOf(match.opponents, function(opponent)
+									return not Table.deepEquals(opponent, slot.opponent)
+								end)
```

Comment:
don't think this works in the following cases:
- team opponent: team switches team template mid GS (e.g. logo change during the GS)
- solo/duo/... opponent: one player in the opponent switches faction or something else that is stored in the opponent
- all opponents: isn't the score set in one while not in the other so the `not Table.deepEquals(opponent, slot.opponent)` would always be true?!

Reply:
MAde a new solution, you think it's good?

Reply:
should work for the start, we can adjust if necessary later on
(possibly for team opponents we will have to do a check against all pagenames set in the team sub-templates in case of historical)